.github/CONTRIBUTING.md

```diff
@@ -49,6 +67,9 @@ outlined on that page and do not file a public issue.
 * 4 spaces for indentation rather than tabs
 * Public classes and methods must contain header documentation
 * Use plain C functions whenever possible (as opposed to class methods)
+* Restrict subclassing (objc macro, final in examples)
```

Comment:
- Can you put the actual macro in here?
- Markdown code formatting on macro and `final`
- How about adding

> except when the class is designed for subclassing